### DISCUSSION TITLE
Dry-Run not showing any info

### DISCUSSION THREAD

**Discussion by lefmannetje** [cecobask/imdb-trakt-sync#69] [Q&A]
I'm trying to do the sync within Github. I got everything setup and it seems to run. But the problem is the sync is done after 52 seconds and I don't see any possible changes. I have over 2000 movies rated on IMDB. I have no lists, just rated movies default.
On Trakt, I have 500 movies rated, so I assumed I would get a return of 1500-ish movies he would like to modify. Is there anything I have to fill in more?

I have these repository secrets:
IMDB_AUTH (cookies)
IMDB_COOKIEATMAIN
IMDB_COOKIEUBIDMAIN
IMDB_EMAIL
IMDB_PASSWORD
SYNC_MODE dry-run
TRAKT_CLIENTID
TRAKT_CLIENTSECRET
TRAKT_EMAIL
TRAKT_PASSWORD

{"time":"2024-12-14T01:42:57.234539087Z","level":"INFO","source":{"function":"github.com/cecobask/imdb-trakt-sync/pkg/client.NewIMDbClient","file":"/home/runner/work/imdb-trakt-sync/imdb-trakt-sync/pkg/client/imdb.go","line":91},"msg":"launched new browser instance","url":"ws://127.0.0.1:43629/devtools/browser/82537ea6-2c03-4455-aea4-c409abf00f72","headless":true,"trace":false}
{"time":"2024-12-14T01:43:06.120308407Z","level":"INFO","source":{"function":"github.com/cecobask/imdb-trakt-sync/pkg/client.(*IMDbClient).hydrate","file":"/home/runner/work/imdb-trakt-sync/imdb-trakt-sync/pkg/client/imdb.go","line":228},"msg":"hydrated imdb client","username":"Lefmannetje","userID":"ur58456915","watchlistID":"ls076097739","lists":[]}
{"time":"2024-12-14T01:43:06.765788729Z","level":"WARN","source":{"function":"github.com/cecobask/imdb-trakt-sync/pkg/client.(*TraktClient).doRequest","file":"/home/runner/work/imdb-trakt-sync/imdb-trakt-sync/pkg/client/trakt.go","line":307},"msg":"trakt rate limit reached, waiting for 3s then retrying http request POST https://trakt.tv/activate/authorize"}
{"time":"2024-12-14T01:43:09.90420595Z","level":"INFO","source":{"function":"github.com/cecobask/imdb-trakt-sync/internal/syncer.(*Syncer).Sync","file":"/home/runner/work/imdb-trakt-sync/imdb-trakt-sync/internal/syncer/syncer.go","line":62},"msg":"sync started"}
{"time":"2024-12-14T01:43:09.90424875Z","level":"INFO","source":{"function":"github.com/cecobask/imdb-trakt-sync/internal/syncer.(*Syncer).syncLists","file":"/home/runner/work/imdb-trakt-sync/imdb-trakt-sync/internal/syncer/syncer.go","line":178},"msg":"skipping watchlist sync"}
{"time":"2024-12-14T01:43:09.904257897Z","level":"INFO","source":{"function":"github.com/cecobask/imdb-trakt-sync/internal/syncer.(*Syncer).syncRatings","file":"/home/runner/work/imdb-trakt-sync/imdb-trakt-sync/internal/syncer/syncer.go","line":236},"msg":"skipping ratings sync"}
{"time":"2024-12-14T01:43:09.904264309Z","level":"INFO","source":{"function":"github.com/cecobask/imdb-trakt-sync/internal/syncer.(*Syncer).syncHistory","file":"/home/runner/work/imdb-trakt-sync/imdb-trakt-sync/internal/syncer/syncer.go","line":269},"msg":"skipping history sync"}
{"time":"2024-12-14T01:43:09.904272755Z","level":"INFO","source":{"function":"github.com/cecobask/imdb-trakt-sync/internal/syncer.(*Syncer).Sync","file":"/home/runner/work/imdb-trakt-sync/imdb-trakt-sync/internal/syncer/syncer.go","line":79},"msg":"sync completed"}

**Comment by cecobask** [comment]
Hi @lefmannetje, the syncer works as expected with the configuration you have provided to it.
Sync of watchlist, ratings and history is disabled by default. You have to tweak the configuration to suit your needs.
For example, if you you'd like to enable sync of ratings, you should set the `SYNC_RATINGS` repository secret to `true`.
More info about the sync configuration can be found here: https://github.com/cecobask/imdb-trakt-sync/blob/main/config.yaml
